queries.yaml

```diff
@@ -328,13 +328,13 @@ post_process:
   - fn: TileStache.Goodies.VecTiles.transform.drop_features_where
     params:
       source_layer: landuse
-      start_zoom: 13
+      start_zoom: 6
       where: >-
         properties.get('mz_is_building') is True
   - fn: TileStache.Goodies.VecTiles.transform.drop_properties
     params:
       source_layer: landuse
-      start_zoom: 13
+      start_zoom: 6
```

Comment:
Agree: that's covered in https://github.com/mapzen/vector-datasource/issues/622.